react/finger-exercises/redux/02 - react-redux/src/app/screens/App/index.js

```diff
@@ -58,21 +62,44 @@ class App extends Component {
         <Navbar />
         <div className={styles.container}>
           <Search onSearch={this.onSearch} />
-          {this.state.books.length ? (
-            this.state.books.map(this.renderBooks)
+          {this.props.books.length ? (
```

Comment:
I'd destructure props before the return so that the code is cleaner, what do you think?